routes/folders.js

```diff
@@ -0,0 +1,91 @@
+const express = require('express')
+const mongoose = require('mongoose')
+
+const router = express.Router()
+
+const {MONGODB_URI} = require('../config')
+
+const Folder = require('../models/folder')
+
+/* ========== GET/READ ALL ITEMS ========== */
+router.get('/', (req, res, next) => {
+  return Folder.find()
+    .sort({name: 'desc'})
+    .then(results => {
+      res.json(results)
+    })
+    .catch(err => {
+      console.error(err)
+      console.error(`ERROR: ${err.message}`)
+      res.sendStatus(500)
```

Comment:
Should call `next(err)` here so all the errors flow thru the error handler, log to the console and respond with JSON
```suggestion
    // res.sendStatus(500) 
    next(err)
```